tfjs-core/src/ops/from_pixels_async_test.ts

```diff
@@ -0,0 +1,232 @@
+/**
+ * @license
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * =============================================================================
+ */
+
+import * as tf from '../index';
+import {BROWSER_ENVS, describeWithFlags} from '../jasmine_util';
+import {expectArraysClose, expectArraysEqual} from '../test_util';
+
+describeWithFlags('fromPixelsAsync', BROWSER_ENVS, () => {
```

Comment:
One issue here is that I don't know how to add test case to check the wrong input and catch the exception. Like https://github.com/tensorflow/tfjs/blob/master/tfjs-core/src/ops/from_pixels_test.ts#L248. @qjia7 @annxingyuan @kainino0x  Do you have any suggestions?